modules/acm/variables.tf

```
@@ -24,6 +24,12 @@ variable "project_id" {
   type        = string
 }
 
+variable "hub_project_id" {
```

I think `fleet` might be more clear for the usage, match `enable_fleet_registration`, etc.

```suggestion
variable "fleet_project_id" {
```

---

I agree with you and changed it. :-)